### PR TITLE
[issue#564] Hide/Unhide Sidebar

### PR DESCRIPTION
**Pull request by dongchime**
### *I made a hamburger button for hiding the sidebar completely and made a "x" button for hiding the sidebar on the top of the sidebar.*

_**examples/twittermap/web/app/views/twittermap/index.scala.html**_

1. Added a hamburger button div with class name "w3-teal"
2. Added a <a> tag to close the sidebar.
3. Added a javascript code to hide/show the sidebar.

**_examples/twittermap/web/public/javascripts/sidebar/controllers.js_**

1. Deleted the function "showOrHideSidebar" that is not necessary.

**_examples/twittermap/web/public/stylesheets/main.css_**

1. #img1, #img2, #img3 is for resizing the map image in sidebar.
2. .w3-teal is for a hamburger button to show the sidebar.
3. .closebtn is for a close button to hide the sidebar.

**Tab the image to play the video demo**
[Everything Is AWESOME](https://youtu.be/o0WXXVox9l8 "Everything Is AWESOME")

**images**
screen shot 2018-08-07 at 4 00 05 pm

screen shot 2018-08-07 at 4 00 09 pm

### CODE REVIEW

**Comment by chenlica** [comment]
LGTM.  @baiqiushi will do the review.

**Comment by chenlica** [comment]
Also for our future PR, do we know how to enable/disable the button?  We don't need this enabling/disabling now, but later.

**Comment by baiqiushi** [comment]
Please take care of the failed building problems.

File: examples/twittermap/web/app/views/twittermap/index.scala.html [hunk 1]
```diff
@@ -32,18 +32,24 @@
...
+  </div>
+
+  <div id='sidebar' class="open" ng-init="click = -1" ng-class="{toggled: click === 1}" ng-controller="SidebarCtrl">
     <div class = "col-xs-2">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeRightMenu()">&times;</a>
       <ul class="nav nav-tabs tabs-left">
         <li role="presentation" ng-click="click = 1; showTab('hashtagTab')"><a href="#hashtag" data-toggle="tab"><i class="fa fa-hashtag fa-2x" aria-hidden="true"></i></a></li>
         <li role="presentation" ng-click="click = 1; showTab('sampletweetTab')"><a href="#tweet" data-toggle="tab"><i class="fa fa-twitter fa-2x" aria-hidden="true"></i></a></li>
         <li role="presentation" ng-click="click = 1; showTab('aboutTab')" class="active"><a href="#about" data-toggle="tab"><i class="fa fa-info-circle fa-2x"></i></a></li>
-        <li role="presentation" ng-click="click = click*-1; showOrHideSidebar(click)"><i ng-class="click ===-1 ? 'fa fa-chevron-left fa-2x': 'fa fa-chevron-right fa-2x'"></i></li>
```
Comment (baiqiushi) [inline examples/twittermap/web/app/views/twittermap/index.scala.html]: We need to call this `showOrHideSidebar()` function when the `hamburger` button is clicked, we can call it inside your `openRightMenu()` function.

Comment (dongchime) [reply examples/twittermap/web/app/views/twittermap/index.scala.html]: Yeah. I fixed it!

**Pull request merged**